pkgs/development/python-modules/opencensus-ext-azure/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, azure-core
+, azure-identity
+, opencensus
+, psutil
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "opencensus-ext-azure";
+  version = "1.1.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-UHYIt36djqq2/9X/EbfOuH9U5KapQMS4FDMbTRsDQVE=";
+  };
+
+  propagatedBuildInputs = [
+    azure-core
+    azure-identity
+    opencensus
+    psutil
+    requests
+  ];
+
+  pythonImportsCheck = [ "opencensus.ext.azure" ];
+
+  meta = with lib; {
+    homepage = "https://github.com/census-instrumentation/opencensus-python/tree/master/contrib/opencensus-ext-azure";
+    description = "OpenCensus Azure Monitor Exporter";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ ];
```

Comment:
same here

Reply:
I'm OK with introducing myself as a maintainer for qcodes modules as I expect to use it in the lab for some time, but I'm not sure I'm the right person to be responsible for the dependencies I do not really know much about...

Reply:
I could, though I don't know much about azure, and I personally won't be using this package, so it would be difficult for me to keep maintaining it.
Maybe we could maintain it together?

Reply:
Sure, two maintainers who have no idea about the package are better then one ^^ Deal =)